tests/unit/acl-v2.tcl

```diff
@@ -258,6 +258,38 @@ start_server {tags {"acl external:skip"}} {
         assert_equal "This user has no permissions to access the 'write1' key" [r ACL DRYRUN command-test GEORADIUS write1 longitude latitude radius M STORE write2]
     }
 
+    test {Test general keyspace commands require some type of permission to execute} {
+        assert_equal "OK" [r ACL DRYRUN command-test touch read]
+        assert_equal "OK" [r ACL DRYRUN command-test touch write]
+        assert_equal "OK" [r ACL DRYRUN command-test touch rw]
+        assert_equal "This user has no permissions to access the 'nothing' key" [r ACL DRYRUN command-test touch nothing]
+
+        assert_equal "OK" [r ACL DRYRUN command-test exists read]
+        assert_equal "OK" [r ACL DRYRUN command-test exists write]
+        assert_equal "OK" [r ACL DRYRUN command-test exists rw]
+        assert_equal "This user has no permissions to access the 'nothing' key" [r ACL DRYRUN command-test exists nothing]
+
+        assert_equal "OK" [r ACL DRYRUN command-test MEMORY USAGE read]
+        assert_equal "OK" [r ACL DRYRUN command-test MEMORY USAGE write]
+        assert_equal "OK" [r ACL DRYRUN command-test MEMORY USAGE rw]
+        assert_equal "This user has no permissions to access the 'nothing' key" [r ACL DRYRUN command-test MEMORY USAGE nothing]
+
+        assert_equal "OK" [r ACL DRYRUN command-test TYPE read]
+        assert_equal "OK" [r ACL DRYRUN command-test TYPE write]
+        assert_equal "OK" [r ACL DRYRUN command-test TYPE rw]
+        assert_equal "This user has no permissions to access the 'nothing' key" [r ACL DRYRUN command-test TYPE nothing]
+    }
+
+    test {Cardinality commands require some type of permission to execute} {
+        set commands {PFCOUNT STRLEN HLEN LLEN SCARD ZCARD XLEN PFCOUNT}
```

Comment:
maybe also add some similar test for HEXISTS and SISMEMBER?

p.s. we flagged HEXISTS differently than SISMEMBER.
i suppose it's because in hash the value is the one attached to the field, and not the field itself?